src/js/base/compat.js

```diff
@@ -93,6 +93,8 @@ if (
         fetch = window.fetch;
     } else if (typeof global !== "undefined" && typeof global.fetch !== "undefined") {
         fetch = global.fetch;
+    } else if (typeof global !== "undefined") {

```

Comment:
We already add a decision bracket (the one below this) where it lead to the import of node-fetch. What part of that condition is failing?

Reply:
`__webpack_require__` is defined

Reply:
move this as a last branch of the chain of "ifs" and "else ifs". When doing so, the if condition must someshow reflect that we're on a nuxt environmennt (using a var or something)